shell/common/switches.cc

```diff
@@ -60,6 +60,7 @@ static const std::string gDartFlagsWhitelist[] = {
     "--lazy_async_stacks",
     "--trace-reload",
     "--trace-reload-verbose",
+    "--enable-service-port-fallback",
```

Comment:
nit: Shall we alphabetize this list? It's getting a bit long.

Reply:
Based on my understanding of the English alphabet, this is now done